app/src/main/java/com/example/helloworld/MainActivity.kt

```diff
@@ -28,9 +28,9 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.activity_main)
 
         auth=FirebaseAuth.getInstance()
-        var Login=findViewById<Button>(R.id.button)
```

Comment:
@raghav32 avoid using findViewById, as we can directly use xml ids as view in Kotlin file

like: "button.text = "Proceed"